executor.go

```diff
@@ -1213,6 +1215,98 @@ func (e *executor) executeClearRowShard(_ context.Context, index string, c *pql.
 	return changed, nil
 }
 
+// executeSetRow executes a SetRow() call.
+func (e *executor) executeSetRow(ctx context.Context, index string, c *pql.Call, shards []uint64, opt *execOptions) (bool, error) {
+	// Ensure the field type supports SetRow().
+	fieldName, err := c.FieldArg()
+	if err != nil {
+		return false, errors.New("SetRow() argument required: field")
+	}
+	field := e.Holder.Field(index, fieldName)
+	if field == nil {
+		return false, ErrFieldNotFound
+	}
+
+	switch field.Type() {
```

Comment:
I would prefer an if for a one state check
 ` if field.Type() != FieldTypeSet{
  return false, fmt.Errorf("SetRow() is not supported on %s field types", field.Type())
}`

Reply:
Yeah, this makes sense. I left it this way in the case that I was going to modify this PR to include the `time` field type.